src/components/admin/EditEventForm/index.tsx

```diff
@@ -141,14 +105,24 @@ const EditEventForm: React.FC<EditEventFormProps> = (props) => {
               onBlur={handleBlur}
             />
           </Form.Item>
-          <Form.Item label="Committee">
-            <Input
-              name="committee"
-              className="input-box"
+          <Form.Item className="committee-wrapper" label="Community">
+            <Select
+              showSearch
+              className="committee-box"
+              size="large"
+              optionFilterProp="children"
               value={values.committee}
-              onChange={handleChange}
-              onBlur={handleBlur}
-            />
+              onChange={(value: string) => setFieldValue('committee', value)}
+              onBlur={() => setFieldTouched('committee', true)}
+              filterOption={(input, option) => {
+                const suborg: string = option.props.children as string;
+                return suborg.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+              }}
+            >
+              {suborgs.map((suborg) => (
+                <Option value={`${suborg}`}>{suborg}</Option>
+              ))}
+            </Select>
```

Comment:
Purely a nitpick, but you _could_ add a `Co-authored-by: StormFireFox1 <matei@gard.us>` to your commit here since this is technically from my other PR (which admittedly was not 100% complete).

This is 100% just me ranting about idiomatic Git practices, so evidently a nitpick in our case.

Reply:
If you show me how, I'd be happy to give you credit for the corresponding commit / for this PR.

Reply:
It's too late now, but the idea is you add a line of this format:

`Co-authored-by: GitHub Username <email@example.com>`

In the commit message of the commit you are crediting for the co-author. If you want to, you can amend that commit with an interactive rebase and force a push, but that's too much effort for a minimal thing. It's fine now, but useful note for the future.

There are others, too, such as:
```
Signed-off-by: # for legal purposes, check the Developer Certificate of Origin on Google
Reported-by: # for bugfixes, crediting the bug reporter
Co-authored-by
Improved-by
Acked-by: # mention in commit or content itself was acknowledged
Tested-by
Reviewed-by
```